### DISCUSSION TITLE
Groups with more than 1500 members

### DISCUSSION THREAD

**Discussion by sivaboseomf** [dsbenghe/Novell.Directory.Ldap.NETStandard#234] [General]
I am trying to get the list of members from a group. If the group has more than 1500 members, it is always returning members 0-1499. How do I get the full list of members?

![ad_search](https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/assets/119540109/485eb97d-960f-49c3-b521-9fe867679d56)

**Comment by dsbenghe** [comment]
Check https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/issues/36
